music/scripts/generate-docs.sh

```diff
@@ -16,14 +16,24 @@
 
 # Regenerates the docs
 #
-# To run, execute 'yarn doc'.
+# To run, execute 'yarn docs'.
+
+# Exit on error.
+set -e
 
 # Generate the docs.
```

Comment:
technically this comment should go above the npx line, these first lines are just setting up variables